ui/pages/import-tokens/import-tokens.js

```diff
@@ -0,0 +1,668 @@
+  ButtonPrimary,
+  ButtonSecondary,
+} from '../../components/component-library';
+import {
+  TextVariant,
+  TextAlign,
+  Severity,
+  TextColor,
+  FontWeight,
+  Size,
+} from '../../helpers/constants/design-system';
+import { Tab, Tabs } from '../../components/ui/tabs';
+import { SECURITY_ROUTE, DEFAULT_ROUTE } from '../../helpers/constants/routes';
+import TokenSearch from '../../components/app/import-token/token-search';
+import TokenList from '../../components/app/import-token/token-list';
+import ZENDESK_URLS from '../../helpers/constants/zendesk-url';
+import { isValidHexAddress } from '../../../shared/modules/hexstring-utils';
+import { addHexPrefix } from '../../../app/scripts/lib/util';
+import { STATIC_MAINNET_TOKEN_LIST } from '../../../shared/constants/tokens';
+import {
+  checkExistingAddresses,
+  getURLHostName,
+} from '../../helpers/utils/util';
+import { tokenInfoGetter } from '../../helpers/utils/token-util';
+import { getPendingTokens } from '../../ducks/metamask/metamask';
+import ImportTokensConfirmation from '../import-tokens-confirmation';
+
+const ImportTokens = () => {
+  const t = useContext(I18nContext);
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const rpcPrefs = useSelector(getRpcPrefsForCurrentProvider);
+  const [mode, setMode] = useState('');
+
+  const [searchResults, setSearchResults] = useState([]);
+  const [tokenSelectorError, setTokenSelectorError] = useState(null);
+  const [selectedTokens, setSelectedTokens] = useState({});
+  const tokenList = useSelector(getTokenList);
+  const [customAddress, setCustomAddress] = useState('');
+  const [customAddressError, setCustomAddressError] = useState(null);
+  const [nftAddressError, setNftAddressError] = useState(null);
+  const [symbolAutoFilled, setSymbolAutoFilled] = useState(false);
+  const [decimalAutoFilled, setDecimalAutoFilled] = useState(false);
+  const [mainnetTokenWarning, setMainnetTokenWarning] = useState(null);
+  const [customSymbol, setCustomSymbol] = useState('');
+  const [customSymbolError, setCustomSymbolError] = useState(null);
+  const [customDecimals, setCustomDecimals] = useState(0);
+  const [customDecimalsError, setCustomDecimalsError] = useState(null);
+  const [tokenStandard, setTokenStandard] = useState(TokenStandard.none);
+  const [forceEditSymbol, setForceEditSymbol] = useState(false);
+  const currentNetwork = useSelector(getCurrentNetwork);
+  const testNetworkBackgroundColor = useSelector(getTestNetworkBackgroundColor);
+  const selectedAddress = useSelector(getSelectedAddress);
+  const isMainnet = useSelector(getIsMainnet);
+  const mostRecentOverviewPage = useSelector(getMostRecentOverviewPage);
+  const identities = useSelector(getMetaMaskIdentities);
+  const tokens = useSelector((state) => state.metamask.tokens);
+  const pendingTokens = useSelector(getPendingTokens);
+  const chainId = useSelector(getCurrentChainId);
+  const theme = useSelector((state) => getTheme(state));
```

Comment:
This can be:

```
const theme = useSelector(getTheme);
```

Reply:
Removed the theme usage, no longer needed after using same color for both dark and light theme